### DISCUSSION TITLE
updated username but classroom uses old username

### DISCUSSION THREAD

**Discussion by daniel-keefe** [community/community#137076] [GitHub Education]
### Hi everyone,

I'm having trouble working with github classroom, and I think the issue must be because I have updated my github username since the last time I taught with it because, searching around, I see some evidence that other folks have experienced a similar issue in the past, e.g.: [Can't Create Classroom #2278](https://github.com/github-education-resources/classroom/issues/2278).  It looks like these have been resolved in the past by github staff.  Could you help me with this as well?

In my case, my current username is "daniel-keefe" but when I go to glassroom.github.com it thinks I am "dfkumn", which is my old username.

**Comment by davecheng-tech** [comment]
I have a similar issue. I changed my GitHub username because I changed workplaces. 

I used GitHub Classroom last year under the username `davecheng-ste`. After recently changing my username to the new `davecheng-tech`, I can see the updated reflected in most places (e.g. `https://github.com/settings/profile` page, my public profile, etc.) however the GitHub Classroom landing page (`https://classroom.github.com/classrooms`) shows me signed in as the older username, `davecheng-ste`, under the profile icon at the top right.

I think this is server side and I cannot find any other place to change or update my username.

Did you end up finding a solution?
